### DISCUSSION TITLE
add support for chiaki remote play ps4/ps5

### DISCUSSION THREAD

**Discussion by iTsMute** [Valkirie/HandheldCompanion#362] [Ideas]
Sdt recently did this.  I think it be a great idea for HC to support chiaki to not just the official remote play app. Options are always good to have

**Comment by CasperH2O** [comment]
@iTsMute what's missing? To my knowledge some stuff doesnt work because Chiaki doesn't have them.

**Reply by iTsMute** [reply]
Not entirely sure myself. I just know one person had reported it not working with HC at all in the past in OKG group. Ill test myself tonight or tomorrow and let you know my findings. 

As far as you know. What doesn't work that chiaki doesn't have? Just so im aware of those beforehand.

Edit: ah I see gyro and touchpads. I wonder if the touchpads got updated recently they seem to work on sdt now apparently.

**Comment by Valkirie** [comment]
We're emulating a 1:1 Dualshock 4 controller. As soon as chiaki complete its DS4 support, it should just work. I believe someone is working on bringing both touchpad and Gyro support to chiaki.

You might want to try 1.3.0 and later builds.

**Reply by iTsMute** [reply]
I'll give it a swing and report back my findings. 

Also I want to clarify I was wrong in my understanding that sdt had it working. "I haven't tested that yet". I took them saying the trackpads was for that as it was already tested and working. So it may not even be working on that either afaik....

**Reply by iTsMute** [reply]
It works. Haven't got to test gesture out yet though.
